CONTRIBUTING.md

```diff
@@ -3,43 +3,70 @@ Pull requests which drastically alter the structure or architecture of the code
 This is to maintain "sanity" whilst going back through code, comparing to the original MIPS disassembled output.
 Best pull requests are which resolve bugs, finish existing decompiled methods, improve readability, introduce new decompiled methods and improve code documentation.
 
-Also it's good to see which issues have more priority to work with here:
-https://github.com/SoapyMan/REDRIVER2/contribute
+It's also good to see which issues have more priority to work with here:
+https://github.com/OpenDriver2/REDRIVER2/contribute
 
-# Compiling
-Project **must** be built only for **32 bit** platform because there are 32 bit pointers everywhere (and use of *long* which with GCC can be a 64 bit numbers)
+#### Markers for decompiled function names, code, etc
 
-First use [premake5](https://github.com/premake/premake-core/releases/download/v5.0.0-alpha15/premake-5.0.0-alpha15-windows.zip) to generate project files.
+- [D] = decompiled
+- [T] = debugged and tested
+- [A] = altered code - debugging purposes or not being able to compile or run.... Can be used anywhere in the code, structures etc
 
-Dependencies (links for Windows):
-- [SDL 2](https://www.libsdl.org/release/SDL2-devel-2.0.12-VC.zip)
-- [OpenAL-Soft](https://openal-soft.org/openal-binaries/openal-soft-1.20.1-bin.zip)
-- [JPEG](http://www.ijg.org/files/jpegsr9d.zip)
+# Base Versions
```

Comment:
I think decompilation information still should be placed somewhere else but not in the top